examples/amp-story/ampconf.html

```diff
@@ -284,20 +284,6 @@ <h1 class="bold">Improve conversions with fast, user-friendly experiences</h1>
           </amp-video>
         </amp-story-grid-layer>
       </amp-story-page>
-
-      <amp-story-page id="page-12">
-        <amp-story-grid-layer template="fill">
-          <amp-img width="400" height="750" layout="fill" src="./img/roadshow.jpg"></amp-img>
-        </amp-story-grid-layer>
-        <amp-story-grid-layer template="thirds" class="bottom">
-          <h1 grid-area="middle-third" class="bold">Join the worldwide AMP Roadshow</h1>
-        </amp-story-grid-layer>
-        <amp-story-cta-layer>
-          <a href="https://www.ampproject.org/amp-roadshow/" class="button medium center">
-            <p class="twenty-px">Sign up Now</p>
-          </a>
-        </amp-story-cta-layer>
-      </amp-story-page>
```

Comment:
It's okay to keep this page! we just need to remove the `cta-layer` wrapper :)
It could use an `amp-story-page-outlink` like you did in the other examples.

Reply:
nice catch. Done!